packages/nimble-components/src/table/components/row/template.ts

```diff
@@ -6,7 +6,7 @@ import { TableCell } from '../cell';
 
 // prettier-ignore
 export const template = html<TableRow>`
-    <template role="row">
+    <template role="row" style="grid-template-columns: inherit;">
```

Comment:
If we need the style then it should be in the stylesheet instead, ie this is a `:host` style.

But actually we shouldn't need it anymore, now that the grid columns are configured as a CSS custom property and CSS custom properties flow through shadow roots, you can use the property set in the higher-level table directly in the stylesheet of this child element, you don't need to do this `inherit` forwarding stuff.